app/controllers/application_controller.rb

```diff
@@ -4,7 +4,7 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
-  before_action :set_honeybadger_context
+                before_action       :set_honeybadger_context
```

Comment:
Inconsistent indentation detected.<br>Put one space between the method name and the first argument.